pandas/tests/groupby/test_groupby.py

```diff
@@ -1320,7 +1320,7 @@ def convert_fast(x):
 
     def convert_force_pure(x):
         # base will be length 0
-        assert len(x.values.base) > 0
+        # assert len(x.values.base) > 0
```

Comment:
delete?

Reply:
Wanted to ask if this test is accurate. IDK what it does right now.

With current behavior, x.values is a view of the sorted DF (which is a copy of the original DF), so it passes.

It fails with this PR, since we're not materializing a sorted DF copy when sorted.

Do you know what this is supposed to check for?

Reply:
no idea, id have to dig into the git history